src/Microsoft.OpenApi/Models/RuntimeExpressionAnyWrapper.cs

```diff
@@ -16,6 +16,20 @@ public class RuntimeExpressionAnyWrapper : IOpenApiElement
         private IOpenApiAny _any;
         private RuntimeExpression _expression;
 
+        /// <summary>
+        /// Parameterless constructor
+        /// </summary>
+        public RuntimeExpressionAnyWrapper() {}
+
+        /// <summary>
+        /// Initializes a copy of an <see cref="RuntimeExpressionAnyWrapper"/> object
+        /// </summary>
+        public RuntimeExpressionAnyWrapper(RuntimeExpressionAnyWrapper runtimeExpressionAnyWrapper)
+        {
+            Any = runtimeExpressionAnyWrapper.Any;
+            Expression = runtimeExpressionAnyWrapper.Expression;
```

Comment:
I believe this needs to have a copy constructor also.

Reply:
The Expression property derives from `RuntimeExpression` that doesn't have a copy constructor as it only has a readonly property `Expression`, hence why I've defaulted to a using a reference copy.

Reply:
Yes, I had forgotten that Expression is an immutable object and so we can copy a reference to it.